File: TicketApp.Server/Models/AppStat.cs
using System.ComponentModel.DataAnnotations;

namespace TicketApp.Server.Models;

public enum AppStatType
{
    TicketsOpened,
    TicketsClosed,
    TicketsClosedLate,
    ChatsSent,
    UsersRegistered
}

public class AppStat
{
    [Key] 
    public AppStatType AppStatType { get; init; }
    
    public int Value { get; set; }
}

File: ticketapp.client/src/models/AppStat.ts
export enum AppStatType
{
    TicketsOpened,
    TicketsClosed,
    TicketsClosedLate,
    ChatsSent,
    UsersRegistered
}

export interface AppStat
{
    appStatType: AppStatType,
    value: number
}

export function GetAppStatOfType(type: AppStatType, appStats?: AppStat[]) : AppStat | undefined
{
    let val: AppStat | undefined = undefined;
    appStats?.map(x => {
        if (x.appStatType == type)
        {
            val = x;
        }
    })
    return val;
}